pom.xml

```diff
@@ -94,10 +95,12 @@
         <version>${nukleus.plugin.version}</version>
         <configuration>
           <scopeNames>mqtt</scopeNames>
+          <packageName>org.reaktivity.specification.mqtt.internal.types</packageName>
         </configuration>
         <executions>
           <execution>
             <goals>
+              <goal>generate</goal>
               <goal>validate</goal>
             </goals>
           </execution>
```

Comment:
Remember to update configuration of the `jacoco` plugin to exclude to code coverage analysis of generated code.
```
      <plugin>
        <groupId>org.jacoco</groupId>
        <artifactId>jacoco-maven-plugin</artifactId>
        <version>0.8.2</version>
        <configuration>
          <excludes>
            <exclude>org/reaktivity/specification/mqtt/internal/types/**/*.class</exclude>
          </excludes>
          <rules>
            <rule>
              <element>BUNDLE</element>
              <limits>
                <limit>
                  <counter>INSTRUCTION</counter>
                  <value>COVEREDRATIO</value>
                  <minimum>${jacoco.coverage.ratio}</minimum>
                </limit>
                <limit>
                  <counter>CLASS</counter>
                  <value>MISSEDCOUNT</value>
                  <maximum>${jacoco.missed.count}</maximum>
                </limit>
              </limits>
            </rule>
          </rules>
        </configuration>
        <executions>
          <execution>
            <goals>
              <goal>prepare-agent</goal>
              <goal>check</goal>
            </goals>
          </execution>
        </executions>
      </plugin>
```